### PR TITLE
tracers: fix err in 4byte, add unigram/bigram/trigram opcode analysis…

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes
This PR is derived from https://github.com/ethereum/go-ethereum/pull/16954.
- It fixes an error in the `4byteTracer`.
- It adds `unigram_tracer.js` which just counts opcodes.
- It adds `bigram_tracer.js` which counts bigrams.
- It adds `trigram_tracer.js` which counts trigrams.

New Tracers Output
* unigram_tracer.js
image
* bigram_tracer.js
image
* trigram_tracer.js
image

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/tracers/internal/tracers/bigram_tracer.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The go-ethereum Authors
```
Comment (KimKyungup) [inline node/cn/tracers/internal/tracers/bigram_tracer.js:1]: Please change license descriptions.
You can refer other file license.

**Comment by KimKyungup** [comment]
@yoomee1313 Please check and show the result of new tracers.

File: node/cn/tracers/internal/tracers/bigram_tracer.js (hunk 1 above)
Comment (yoomee1313) [reply node/cn/tracers/internal/tracers/bigram_tracer.js:2]: I change license descriptions of tracer javascript files. Please check those changes @KimKyungup

**Comment by yoomee1313** [comment]
@KlaytnDev I added the results of new tracers. Also, I changed descriptions of tracers in this commit (https://github.com/klaytn/klaytn/pull/752/commits/98a1ae1f76da15ca5ca92d5e0f5ee1642f671d48).

**Review by ehnuje — DISMISSED** [review]

File: node/cn/tracers/internal/tracers/bigram_tracer.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Modifications Copyright 2020 The klaytn Authors
```
Comment (jeongkyun-oh) [inline node/cn/tracers/internal/tracers/bigram_tracer.js:1]: I think the license should be updated when we modified the file. If the file is exactly the same as the one in ethereum, we didn't put the modification comments.

Comment (yoomee1313) [reply node/cn/tracers/internal/tracers/bigram_tracer.js:1]: You're right.. I added klaytn copyright only to the modified files https://github.com/klaytn/klaytn/pull/752/commits/8a8c113dfca481a39163e4dc858039b9f42da3f4, Thanks!

**Review by ehnuje — APPROVED** [review]

**Comment by kjhman21** [comment]
@yoomee1313 @KimKyungup Will this new tracers be introduced in Klaytn Docs, right? 😄

**Comment by yoomee1313** [comment]
@kjhman21 Yes, updating Klaytn Docs(https://docs.klaytn.com/bapp/json-rpc/api-references/debug/tracing#debug_tracetransaction) is necessary when those new tracers are included in release

**Comment by ehnuje** [comment]
> @kjhman21 Yes, updating Klaytn Docs(https://docs.klaytn.com/bapp/json-rpc/api-references/debug/tracing#debug_tracetransaction) is necessary when those new tracers are included in release

@yoomee1313 Not to miss updating the docs, how about making a JIRA item to track the task?

**Comment by yoomee1313** [comment]
> > @kjhman21 Yes, updating Klaytn Docs(https://docs.klaytn.com/bapp/json-rpc/api-references/debug/tracing#debug_tracetransaction) is necessary when those new tracers are included in release
> 
> @yoomee1313 Not to miss updating the docs, how about making a JIRA item to track the task?

@ehnuje Instead of making a JIRA item, i will tag a label to the PRs that need docs!

**Review by ehnuje — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]

**Comment by KimKyungup** [comment]
@yoomee1313 Please update klaytn-docs before merging this.

**Review by winnie-byun — APPROVED** [review]

**Pull request merged**